deploy-board/deploy_board/templates/deploys/deploys.tmpl

```diff
@@ -40,8 +41,9 @@
                 {{ deploy_summary.build|branchAndCommit }}
             </a>
         </td>
+        <td><a href="/clouds/accounts/{{ deploy_summary.account.provider }}/{{ deploy_summary.account.cell }}/{{ deploy_summary.account.id }}">{{ deploy_summary.account.id }}</a></td>
```

Comment:
Let's hide this link if account is `None` for backward compatibility in cases when Deploy Board will be deployed before cluster management service.

Or if deployment was done in the cluster which was provisioned manually.

Reply:
fixed, thanks